samples/aurora-serverless-global-db-cdk/README.md

```diff
@@ -0,0 +1,115 @@
+    ```
+3. Install dependencies:
+    ```
+    npm install
+    ```
+4. Configure AWS CDK to bootstrap the AWS account, primary region and secondary region :
+    ```
+    cdk bootstrap 111111111111/eu-west-1
+    cdk bootstrap 111111111111/eu-west-2
+    ```
+5. From the command line, use AWS CDK to deploy the all the stacks synthesized: 
+    ```
+    cdk deploy --all
+    ```
+    Alternatively you can also deploy each stack individually. From the command line, use AWS CDK to deploy each stack: 
+    ```
+    cdk deploy aurora-global-cluster
+    cdk deploy primary-cluster
+    cdk deploy secondary-cluster
+    cdk deploy primary-test-app
+    cdk deploy secondary-test-app
+    ```
+7. Note the outputs from the CDK deployment process. These contain the primary and secondary URLs which are used for testing.
+
+## How it works
+
+- The template in this sample synthesizes five stacks for deployment using the multiple stack approach of CDK. 
+- The first stack named aurora-global-cluster creates the Aurora Global Cluster. 
+- The second stack named primary-cluster deploys the primary cluster with a serverless v2 instance in the primary region defined.
+- The third stack named secondary-cluster deploys the secondary cluster with a serverless v2 instance in the secondary region defined.
+- The fourth and fifth stack named primary-test-app and secondary-test-app deploys a fargate container with a nodejs app for testing the global database tables. 
+- The primary cluster supports both write and read operations. The secondary cluster supports read operation only.
+- Once you deploy all the stacks, you have built a global database that automatically replicates data from the primary to the secondary region.
+
+## Testing
+
+Note down the primary-test-app.FargateServiceURL and secondary-test-app.FargateServiceURL values from the CDK output and update them in each of the test commands below. 
+
+1.  Initialize the Global Database by creating a table 
+
+    ```
+    curl primary-test-app.FargateServiceURL/init
+    ```
+
+    Expected Response :
+    "Task table created.."
+
+2. Write a task into the Primary Cluster 
+    ```
+    curl -X POST primary-test-app.FargateServiceURL/tasks -H 'Content-Type: application/json' -d '{"name":"Task1","status":"created"}'
+    ```
+    Expected Response : Task added with ID: 1
+3. Read the tasks from the Secondary cluster 
+    ```
+    curl secondary-test-app.FargateServiceURL/tasks 
+    ```
+    Expected Response : 
+    [{"id":1,"name":"Task1","status":"created"}]
+
+4. Attempt to write a task into the Secondary cluster 
+    ```
+    curl -X POST secondary-test-app.FargateServiceURL/tasks -H 'Content-Type: application/json' -d '{"name":"Task1","status":"created"}'
+    ```
+    Expected Response : error : cannot execute INSERT in a read-only transaction
+    
+    Note : You can enable Write forwarding feature to continue to use the Secondary cluster endpoint for write transactions as well. 
```

Comment:
Add link to docs for the write forwarding feature

Reply:
Added